tests/View/Alert.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Performing\TwigComponents\Tests\View;
+
+use Performing\TwigComponents\View\Component;
+
+class Alert extends Component
+{
+    /**
+     * The alert type.
+     *
+     * @var string
+     */
+    public $type;
+
+    /**
+     * The alert message.
+     *
+     * @var string
+     */
+    public $message;
+
+    /**
+     * The alert types.
+     *
+     * @var array
+     */
+    public $types = [
+        'default' => 'text-indigo-50 bg-indigo-400',
+        'success' => 'text-green-50 bg-green-400',
+        'caution' => 'text-yellow-50 bg-yellow-400',
+        'warning' => 'text-red-50 bg-red-400',
+    ];
+
+    protected $counter = 0;
+
+    /**
+     * Create the component instance.
+     *
+     * @param  string  $type
+     * @param  string  $message
+     * @return void
+     */
+    public function __construct($type = 'default', $message = null)
```

Comment:
The requirement to always provide default values for component constructor arguments doesn't seem like a good idea to me. Some props simply don't have a natural default value and forcing one could lead to confusion. Instead, I'd prefer the code to fail hard and tell me that some required argument is missing. :)